apps/dashboard/app/helpers/scripts_helper.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+module ScriptsHelper
+end
```

Comment:
Why is this blank file here?

Reply:
No problem, we likely shouldn't add things when we don't really need it. I've removed it.